app/auth/views.py

```diff
@@ -19,6 +19,10 @@ def before_request():
 					#find the user with the id on the token
 					user = User.query.filter_by(id=res).first()
 					g.user = user
+					return
```

Comment:
an empty return statement

Reply:
yes, in python return statements mean the end of the current process if they return a value. However, if it returns nothing or `None` the process does not stop. 

in this scenario, this is a pre_request hook, this function runs before every request is processed to ensure a user is authenticated. if a user is authenticated, it returns nothing, meaning the app should go ahead and process the request.